source/_integrations/aussie_broadband.markdown

```diff
@@ -0,0 +1,21 @@
+---
+title: Aussie Broadband
+description: Instructions on how to integrate Aussie Broadband within Home Assistant.
+ha_category:
+- Network
+ha_release: 2021.9
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+- '@nickw444'
+ha_domain: aussie_broadband
+ha_platforms:
+- sensor
+- camera
```

Comment:
A new integration can only bring in only a single platform, so this seems to be incorrect?

Reply:
Fair call, I thought the additional platform (camera in this case) was trivial enough, but I'll follow your guidance and strip back the PR in core to be only for `sensor`.

Reply:
```suggestion
- sensor
```